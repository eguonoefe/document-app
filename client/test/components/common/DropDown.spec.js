/* global expect jest test */
import React from 'react';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import DropDown from '../../../components/common/DropDown';

configure({ adapter: new Adapter() });

jest.dontMock('../../../components/common/DropDown');


describe('DropDown Component', () => {
  test('should match the dropdown snapshot', () => {
    const component = shallow(
      <DropDown selected={'active'} value={'Hi'} text={'Hi'} />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
