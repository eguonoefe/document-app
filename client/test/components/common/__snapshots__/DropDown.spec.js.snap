// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropDown Component should match the dropdown snapshot 1`] = `
<option
  selected="active"
  value="Hi"
>
  Hi
</option>
`;
